packages/graphiql/cypress/e2e/tabs.cy.ts

```diff
@@ -64,7 +82,13 @@ describe('Tabs', () => {
       response: { data: { image: '/images/logo.svg' } },
     });
 
-    // Close tab
+    // Close tab (this will get rejected)
+    cy.get('#graphiql-session-tab-1 + .graphiql-tab-close').click();
+
+    // Tab is still visible
+    cy.get('#graphiql-session-tab-1 + .graphiql-tab-close').should('exist');
+
+    // Close tab (this will get accepted)
```

Comment:
it is better to add independent test for confirmation, and not merge it with existent one